ampersand-filter-tracker.js

```diff
@@ -0,0 +1,140 @@
+;(function() {
+  var d3 = require('d3');
+  var _ = require('lodash');
+
+  var AmpersandState = require('ampersand-state');
+  var AmpersandView = require('ampersand-view');
+
+  var FilterTrackerState = AmpersandState.extend({
+    props: {
+      filter: 'function'
+    },
+    session: {
+      handles: 'array'
+    },
+    initialize: function() {
+      _.each(this.handles, function(handle) {
+        console.log(handle.model);
+        handle.active = false;
+        _.each(handle.props, function(prop) {
+          handle.model.on('change:' + prop, function(model, prop) {
```

Comment:
You can update things here so that you are only calling this once (you'll be removing the above _.each) - what do you think?

```javascript
function changeEvt(prop) {
  return 'change:' + prop;
}

handle.model.on(_.map(handle.props, changeEvt).join(' '), function(model, prop) {
```